testsuite/MDAnalysisTests/lib/test_util.py

```diff
@@ -221,6 +221,13 @@ def test_normal(self, vec1, vec2, value):
         assert_equal(mdamath.normal(vec1, vec2), value)
         # add more non-trivial tests
 
+    def test_angle_lower_clip(self):
+    	a = np.array([0.1, 0, 0.2])
```

Comment:
Travis is picking up some tabs instead of spaces on some of these lines. Could you make sure that you are solely using tabs?

Reply:
I hope you mean "spaces" and not "tabs" – please never, ever use tabs in Python code!

Reply:
Sorry yes, that is a very very bad typo. Spaces not tabs!